frontstage/templates/register/register.enter-enrolment-code.html

```diff
@@ -30,7 +29,7 @@ <h1 class="panel__title venus">Enrolment code not valid</h1>
 
         <h1 class="saturn">Create an account</h1>
 
-        {% if errorType == "failed" %}
+        {% if errorType == "failed" or errors|length > 0 %}
```

Comment:
Is there any way to combine `errorType` and `errors` would be nice if all our errors got passed in in the same way

Reply:
I agree with this, although i have no idea where the errorType comes from, will take a look into this

Reply:
So currently not possible to do without putting the IAC checks in the form. Spoke to Lewis, he's keen on keeping the two things separate